Backup.md

```diff
@@ -0,0 +1,29 @@
+## Solr Backup and Restore
+
+When using this backup/restore mechanism, Solr's replication handler is using Caringo Swarm as the storage layer for the backups. The implementation of the new backend repository is [here](https://github.com/xenit-eu/solr-backup).
```

Comment:
this backup/restore script is and should not be tied to using the s3-backup-repository, is it ?

Reply:
It's linked via parameters "repository=swarm&location=swarm:///" in the replication command.

Reply:
so the backup-repository is not pre-configured, but a parameter with the replication command ?

Reply:
which still makes this an orthogonal concern ?

Reply:
Will leave (possible) parametrization of repository & location for a new PR.